library/sql_upgrade_fx.php

```diff
@@ -801,8 +830,20 @@ function upgradeFromSqlFile($filename)
             if ($skipping) {
                 echo "<font color='green'>Skipping section $line</font><br />\n";
             }
-        } // convert all *text types to use default null setting
-        else if (preg_match('/^#IfTextNullFixNeeded/', $line)) {
+        } else if (preg_match('/^#IfNotWenoRx/', $line) && !tableHasRow('erx_weno_drugs', "drug_id", '1008') == true) {
+            if (listExists("erx_weno_drugs")) {
```

Comment:
Is the upgrade function working? The use of listExists does not seem right.
Probably make more sense to remove the listExists call and migrate your above `tableHasRow('erx_weno_drugs', "drug_id", '1008') == true` call above to there instead.